transaction-history-manager/src/main/java/com/qpay/transactionhistorymanager/controller/ReportController.java

```diff
@@ -0,0 +1,26 @@
+package com.qpay.transactionhistorymanager.controller;
+
+import com.qpay.libs.models.ReportInfo;
+import com.qpay.transactionhistorymanager.service.ReportGeneratorService;
+import com.qpay.transactionhistorymanager.utility.PathUtils;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.io.IOException;
```

Comment:
1) after 8 months of work, you still did not setup automatic removal of unused imports? check our wiki on finally do it.
2) after 8 months of work, you still ignore sonar issues and make me review your code manually instead of resolving sonar warnings.

It's really annoying and not funny